browser-interface/packages/renderer-protocol/inverseRpc/services/mutualFriendsService.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  GetMutualFriendsResponse,
+  MutualFriendsKernelServiceDefinition
+} from '@dcl/protocol/out-ts/decentraland/renderer/kernel_services/mutual_friends_kernel.gen'
+import { RpcServerPort } from '@dcl/rpc'
+import * as codegen from '@dcl/rpc/dist/codegen'
+import { RendererProtocolContext } from '../context'
+import { getMutualFriends } from 'shared/friends/sagas'
+
+export function registerMutualFriendsKernelService(port: RpcServerPort<RendererProtocolContext>) {
```

Comment:
I'm not sure that I understand the flow of this registration, who is calling it?

Reply:
It's called from [here](https://github.com/decentraland/unity-renderer/blob/c56c628d39c23d0eeea4b721b0d5b1dbefd148c2/browser-interface/packages/renderer-protocol/inverseRpc/rpcServer.ts#L24-L30), and as far as I understand, it's added to a registration list. Each registered service is then used to map ports to their corresponding function implementations.